Added Location loading screen

saurabhshalu opened the pull request:
When user visit the public room, the full screen overlay loading will be displayed with appropriate message until user gives location access.

Kindly consider merging the PR and close the issue.

Aagam-Jain-113 commented:
@saurabhshalu As reference in #17 , whenever we click deny button we want to display a popup message displaying Ask for location again or back to home button instead of leaving the page as it is, Add even this message in our page .

saurabhshalu commented:
> @saurabhshalu As reference in #17 , whenever we click deny button we want to display a popup message displaying Ask for location again or back to home button instead of leaving the page as it is, Add even this message in our page .

Okay, I'll make the changes tomorrow night, as I have some important college work to finish tomorrow. Sorry for the delayed reply.

saurabhshalu commented:
@Aagam-Jain-113 
I added a custom message with go back button if user denied the location access.
As it is not possible to again show the location popup if denied by the user manually, I added a customized message stating "**enable location from top left corner of address bar**"
There is Go back button also, which will take user to the homepage.

Kindly consider merging the PR.

saurabhshalu commented:
@Aagam-Jain-113 
There are few changes in the labelling of the hacktoberfest rules regarding labelling. 
Kindly go through this link:
https://hacktoberfest.digitalocean.com/hacktoberfest-update

Mark previous pull requests as ‘hacktoberfest-accepted’ too if possible.
